sandstone/lib/handlers/broadcast.py

```diff
@@ -4,10 +4,24 @@
 import json
 from pydispatch import dispatcher
 
+
+class BroadcastManager(object):
+    _clients = set()
```

Comment:
Since we're employing a singleton pattern here, and there is one instance of `BroadcastManager` bound to the `SandstoneApplication`, this should not be defined as a static variable. Instead, `_clients` should be defined within the `__init__()` function for this class.